web/src/main/java/it/pagopa/selfcare/mscore/web/controller/UserController.java

```diff
@@ -221,4 +223,27 @@ public ResponseEntity<Void> deleteProducts(@ApiParam("${swagger.mscore.relations
         userService.findAndUpdateStateByInstitutionAndProduct(userId, institutionId, productId, RelationshipState.DELETED);
         return ResponseEntity.ok().build();
     }
+
+    /**
+     * The function retrieves userInfo given UserId and optional productId
+     *
+     * @param userId                String
+     * @param productId         String
+     * @return onboardingInfoResponse
+     * <p>
+     * * Code: 200, Message: successful operation, DataType: TokenId
+     * * Code: 400, Message: Invalid ID supplied, DataType: Problem
+     * * Code: 404, Message: Not found, DataType: Problem
+     */
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "${swagger.mscore.users}", notes = "${swagger.mscore.users}")
+    @GetMapping(value = "/users/{id}")
+    public ResponseEntity<UserResponse> getUserProductsInfo(@ApiParam("${swagger.mscore.relationship.relationshipId}")
```

Comment:
"swagger.mscore.relationship.relationshipId" should be something like "....users.id"